docs/pyalgotrad/index.md

```diff
@@ -11,76 +10,56 @@
-- Modules:
-    - [constants](module_constants.md)
-
 # pyalgotrading
-
 Official Python Package for Algorithmic Trading APIs powered by AlgoBulls!
 
 ### Features
-
 - Powered by the [AlgoBulls Platform](https://algobulls.com)
 - Everything related to Algorithmic Trading Strategies!
     - Create & upload strategies on the [AlgoBulls Platform](https://algobulls.com)
     - Free pool of Strategies are available separately at [pyalgostrategypool](https://github.com/algobulls/pyalgostrategypool)!
     - Support for all 150+ Technical Indicators provided by [TA-Lib](https://mrjbq7.github.io/ta-lib/index.html)
     - Support for multiple candlesticks patterns - Japanese OHLC, Renko, Heikin-Ashi, Linebreak
-    - Support for multiple candle inntervals - 1 minute, 3 minutes, 5 minutes, 10 minutes, 15 minutes, 30 minutes, 1 hour, 1 day.
+    - Support for multiple candle intervals - 1 minute, 3 minutes, 5 minutes, 10 minutes, 15 minutes, 30 minutes, 1 hour, 1 day. 
     - Support for **Regular Orders** and **Bracket Orders**
     - Support for **MARKET**, **LIMIT**, **STOPLOSS**, **STOPLOSS-LIMIT** orders
     - Support for **INTRADAY** and **DELIVERY** orders
 - Support for **Backtesting**
 - Support for **Paper Trading**
 - Support for **Real Trading**
-- Support for multiple brokers for Real Trading
-    - Currently supported brokers - Zerodha
+- Support for multiple brokers for Real Trading 
+    - Currently supported brokers - Zerodha 
     - Coming soon - AliceBlue, Mastertrust
     - More on the way!
 - Real-time Logs for Backtesting, Paper Trading, Real Trading
-- Multiple real-time Reports available for Backtesting, Paper Trading and Real Trading -
+- Multiple real-time Reports available for Backtesting, Paper Trading and Real Trading - 
     - Profit-n-Loss report (PnL report)
     - Statistics of (PnL report)
     - Order History for each order with state transitions & timestamps
 - Plot Candlestick charts using [plotly.py](https://github.com/plotly/plotly.py)
-
+    
 Backtesting, Paper Trading and Real Trading can be performed on the same strategy code base!
 
 ### Documentation
-
 You can find the docs [here](https://algobulls.github.io/pyalgotrading/).
 
-### Python
-
+### Python 
 - Python Support: `Python 3.6+`.
 - Python Requirements: See [requirements.txt](https://github.com/algobulls/pyalgotrading/blob/master/requirements.txt).
 - We recommend you to use the latest version of Python (v3.8+) to enjoy better performance benefits, especially for pandas. (For Python 3.6, the latest supported Pandas version is v0.25.3. For Python 3.8, the latest supported version is v1.0.x)
 
 ### Installation
-
-Package can be easily installed using `pip` -
-
+Package can be easily installed using `pip` - 
 ```
 pip install pyalgotrading
 ```
 
 ### Support / Getting Help
-
 - *Bug Reporting / New Feature Request*: Please [create a new issue](https://github.com/algobulls/pyalgotrading/issues/new) here on GitHub.
 - *Discussion Forum*: Coming Soon!
```

Comment:
we can add
`https://community.algobulls.com/c/github/pyalgotrading/42`
for the discussion forum till we get something better

Reply:
done